automated/android/microbenchmarks/microbenchmarks.sh

```diff
@@ -51,23 +51,23 @@ fi
 
 mkdir workspace
 cd workspace || exit
+
+# Download/extract the stripped down tree.
+wget  -q "${SNAPSHOTS_URL}/test-tree.txz"
+tar -xf "test-tree.txz"
+
+# Download/extract the build results.
 wget  -q "${SNAPSHOTS_URL}"/"${BUILD_TARBALL}"
-[ -z "$HOME" ] && export HOME="/"
-git config --global user.email "ci_notify@linaro.org"
-git config --global user.name "Linaro CI"
-git config --global --add color.ui auto
 tar -xf "${BUILD_TARBALL}"
+
+[ -z "$HOME" ] && export HOME="/"
 export PATH=${PWD}/out/host/linux-x86/bin/:${PATH}
 
 # FIXME removing latest adb from build since it is not working well from container
 rm -rf "${PWD}"/out/host/linux-x86/bin/adb
 initialize_adb
 adb_root
-curl https://storage.googleapis.com/git-repo-downloads/repo > "${PWD}"/out/host/linux-x86/bin/repo
```

Comment:
Can't we remove 'curl' from PKG_DEPS, since that isn't used anymore what I can see.

Reply:
Good point!
I will do in the next commit!